src/models/utils.ts

```diff
@@ -274,13 +274,13 @@ export function isSameConnection(conn: IConnectionCredentials, expectedConn: ICo
  * Check if a file exists on disk
  */
 export function isFileExisting(filePath: string): boolean {
-        try {
-            fs.statSync(filePath);
-            return true;
-        } catch (err) {
-            return false;
-        }
+    try {
```

Comment:
I'd recommend to send reformatting changes separately from functional changes.  It makes the functional change PRs unnecessarily complex, increasing the likelihood an important change will be missed.

Reply:
These changes came in due to linting before pre-commit, will remove next time